react_native/trimblemaps-mobile-sdk-demo-reactnative/react_native/RNMapsSampleApp/ios/export.plist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Trimble specific signing? If so we should blank these out and maybe add a readme explaining what an integrator would have to do. Other public iOS projects would likely give you some ideas.